scenes/lessons/7th_class/miandam/miandam.py

```diff
@@ -0,0 +1,578 @@
+        surr_rect_five_x_plus_six_y = SurroundingRectangle(five_x_plus_six_y, buff=MED_SMALL_BUFF)
+        surr_rect_six_y = SurroundingRectangle(five_x_plus_six_y[2], color=BLUE, corner_radius=0.25)
+        surr_rect_five_x = SurroundingRectangle(five_x_plus_six_y[0], color=BLUE, corner_radius=0.25)
+        surr_rect_five_x.stretch(1.2, 1).align_to(surr_rect_six_y, UP)
+
+        bazmandam_arrow = Arrow(color=YELLOW).rotate(PI / 4).next_to(surr_rect_five_x_plus_six_y, UR, buff=0)
+        miandam_arrows = VGroup(
+            Arrow(color=BLUE).rotate(- PI * 3 / 8).next_to(surr_rect_five_x, DOWN, buff=0).shift(0.2 * RIGHT),
+            Arrow(color=BLUE).rotate(- PI * 5 / 8).next_to(surr_rect_six_y, DOWN, buff=0).shift(0.2 * LEFT)
+        )
+
+        bazmandam = Tex('Բազմանդամ', tex_template=ARMTEX, font_size=60).next_to(bazmandam_arrow, UR)
+        miandams = Tex('Միանդամներ', tex_template=ARMTEX, font_size=60).next_to(miandam_arrows, DOWN)
+
+
+
+# ANIMATIONS
+
+        self.add(five_x_plus_six_y)
+        self.wait()
+
+        self.play(Create(surr_rect_five_x), Create(surr_rect_six_y))
+        self.wait()
+        self.play(Create(surr_rect_five_x_plus_six_y))
+        self.wait(0.25)
+        self.play(Create(bazmandam_arrow))
+        self.play(Write(bazmandam))
+        self.wait()
+
+        self.play(Create(miandam_arrows))
+        self.play(Write(miandams))
+        self.wait()
+
+        self.play(*[FadeOut(mob) for mob in self.mobjects])
+
+
+class FiveXPlusSixYPlusFourZ(Scene):
+    def construct(self):
+
+# INITS
+
+        five_x_plus_six_y_plus_4_z = Tex('5x', ' + ', '6y', ' + ', '4z', font_size=70).shift(0.5 * LEFT)
+        surr_rect_five_x_plus_six_y_plus_4_z = SurroundingRectangle(five_x_plus_six_y_plus_4_z, buff=MED_SMALL_BUFF)
+        surr_rect_six_y = SurroundingRectangle(five_x_plus_six_y_plus_4_z[2], color=BLUE, corner_radius=0.25)
+        surr_rect_five_x = SurroundingRectangle(five_x_plus_six_y_plus_4_z[0], color=BLUE, corner_radius=0.25)
+        surr_rect_four_z = SurroundingRectangle(five_x_plus_six_y_plus_4_z[4], color=BLUE, corner_radius=0.25)
+        surr_rect_five_x.stretch(1.2, 1).align_to(surr_rect_six_y, UP)
+        surr_rect_four_z.stretch(1.2, 1).align_to(surr_rect_six_y, UP)
+
+        bazmandam_arrow = Arrow(color=YELLOW).rotate(PI / 4).next_to(surr_rect_five_x_plus_six_y_plus_4_z, UR, buff=0)
+        miandam_arrows = VGroup(
+            Arrow(color=BLUE).rotate(- PI * 3 / 8).next_to(surr_rect_five_x, DOWN, buff=0).shift(0.2 * RIGHT),
+            Arrow(color=BLUE).rotate(- PI * 4 / 8).next_to(surr_rect_six_y, DOWN, buff=0),
+            Arrow(color=BLUE).rotate(- PI * 5 / 8).next_to(surr_rect_four_z, DOWN, buff=0).shift(0.2 * LEFT),
+        )
+
+        bazmandam = Tex('Բազմանդամ', tex_template=ARMTEX, font_size=60).next_to(bazmandam_arrow, UR, buff=0.2)
+        miandams = Tex('Միանդամներ', tex_template=ARMTEX, font_size=60).next_to(miandam_arrows, DOWN)
```

Comment:
Are those lessons gonna be in Armenian only?
Just curious.

Reply:
չգիտեմ ։դ լավ կլիներ անգլերենն էլ անեի ամեն դեպքում

Reply:
կարելի է թողնել հաջորդ դասին